src/explore-education-statistics-common/src/modules/find-statistics/components/charts/MapBlock.tsx

```diff
@@ -1,88 +1,657 @@
-import { Feature, GeoJsonProperties, Geometry } from 'geojson';
+import {
+  Feature,
+  FeatureCollection,
+  GeoJsonProperties,
+  Geometry,
+} from 'geojson';
 import 'leaflet/dist/leaflet.css';
-import React, { Component } from 'react';
-import { GeoJSON, Map } from 'react-leaflet';
+import React, { Component, createRef } from 'react';
+import { GeoJSON, LatLngBounds, Map } from 'react-leaflet';
 import { ChartProps } from '@common/modules/find-statistics/components/charts/ChartFunctions';
+import {
+  DataBlockData,
+  DataBlockGeoJsonProperties,
+  DataBlockLocation,
+  DataBlockLocationMetadata,
+  DataBlockMetadata,
+} from '@common/services/dataBlockService';
+import { FormSelect } from '@common/components/form';
+import classNames from 'classnames';
+import { SelectOption } from '@common/components/form/FormSelect';
+import { Dictionary } from '@common/types/util';
+import UKGeoJson from '@common/services/UKGeoJson';
```

Comment:
Yeah... I was wondering how to do that... it's stupidly large

Reply:
You should probably put it in state so that there is a re-render when it imports. Something like this should do the trick (or a promise variant):

```javascript
public async componentDidMount() {
  const { default } = await import('@common/services/UKGeoJson');
  this.setState({ ukGeoJson: default });
}